x-pack/test/functional/services/machine_learning/data_frame_analytics_table.ts

```diff
@@ -80,6 +84,11 @@ export function MachineLearningDataFrameAnalyticsTableProvider({ getService }: F
       );
     }
 
+    public async openResultsView() {
+      await this.assertJobViewButtonExists();
+      await testSubjects.click('mlAnalyticsJobViewButton');
```

Comment:
After a click we should wait for something to happen, such that the next called service method can be sure that everything is ready to go. In this case we want to make sure, that the click really opened the results page:
```
await testSubjects.existOrFail('mlPageDataFrameAnalyticsExploration', { timeout: 5000 });
```

Reply:
Good point - added in 6116a586782001a074d9582df01434f18a3c8975